lib/rubocop/cop/layout/indentation_consistency.rb

```diff
@@ -78,7 +78,7 @@ module Layout
       #   class A
       #     def test
       #       puts 'hello'
-      #        puts 'world'
+      #       puts 'world'
```

Comment:
This is not incorrect, but it is changing to clarify the differences only the access modifiers.